src/main/java/com/microsoft/graph/http/CoreHttpProvider.java

```diff
@@ -374,11 +387,11 @@ public Request authenticateRequest(Request request) {
 									.newBuilder()
 									.connectTimeout(connectionConfig.getConnectTimeout(), TimeUnit.MILLISECONDS)
 									.readTimeout(connectionConfig.getReadTimeout(), TimeUnit.MILLISECONDS)
-									.followRedirects(false)
+									.followRedirects(false) //TODO those are duplicated for the time being with core, remove in 3.0
```

Comment:
Please create the related github issue and link it here.

Reply:
done #516 do you also want me to remove the comment from the code or can we leave it for now?

Reply:
Linking the issue in the comment itself is helpful because the comment cleanup can easily be done by anybody in the team by following the progress of the issue later on.

Otherwise, if the issue gets resolved, the comment will be stale and we won't be able to tell the staleness without going git blame -> open the pull request -> find this comment linking to the issue.

I would be happy to hear if there are alternative approaches. This was what we used to do in our previous teams and it worked quite well.

cc: @MIchaelMainer @ddyett in case they have any input.

P.S. sorry about leaving meta-level comments in your PRs but I am working on standardizing the process for the team.

Reply:
do you mean adding the link to the issue in the code?

Reply:
I think it's a bit "heavy" but I don't see any other process that would circumvent the "issue has been closed with no code changes" scenario.

Reply:
```suggestion
									.followRedirects(false) //TODO https://github.com/microsoftgraph/msgraph-sdk-java/issues/516
```